tests/test_plugins.py

```diff
@@ -134,6 +137,44 @@ def _update_context(st, max_workers, fallback_gains=None):
             'timeout': 60,  # we don't want to build travis for ever
         })
 
+def _test_child_options(st):

```

Comment:
Although this looks like a `strax`-like test I think it will do no harm and do catch potential problems :)